tests/airgapped/knative/README.md

```diff
@@ -0,0 +1,37 @@
+# Testing Knative in airgapped
+
+This directory is dedicated to testing Knative in an airgapped environment.
+
+## The `helloworld` Knative Service example
+The example used for testing is the `helloworld.yaml` Knative Service.
+
+The `knative/helloworld-go` image will be included in your airgapped environment given that you used the [Airgapped test scripts](../README.md). It's specifically added in the [`get-all-images.sh` script](../../../scripts/airgapped/get-all-images.sh).
```

Comment:
I like the information in this paragraph, but I'm thinking that maybe we can have a "Prereqs" section that says this script has to run beforehand or to make sure that the image is present in the local registry. Just because people usually look at prereqs instead of reading all paragraphs.

You could even include the "Prepare the airgapped environment and Deploy CKF by following the steps in [Airgapped test scripts](../README.md)." info.

Reply:
good point, i changed it up a bit, wdyt?